### DISCUSSION TITLE
install and configure wordpress sites behind Nginx Proxy Manager

### DISCUSSION THREAD

**Discussion by Nikerkeee** [NginxProxyManager/nginx-proxy-manager#3591] [Q&A]
Hello guys,

at home, i have installed an ubuntu 22.04 VM and an NPM server installed. they are separated VMs in the same environment.

on the ubuntu server from the local address the wordpress site working correctly, but when im setting up the NPM like this:
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/60218280/6570ceb5-a6de-4346-a070-e3f80422429f)
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/60218280/2ae9e68a-9da7-4b22-ad2a-279c96e4fadd)

and getting the following error:

![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/60218280/2ba07f4f-84d1-4801-87e4-3b93f7bd44f2)

i want to use the [domain.com](https://domain.com/) as [200.0.0.23/creaprint/wp-admin](https://200.0.0.23/creaprint/wp-admin), but i dont know what is the problem with the NPM settings.
The wordpress folder path is : /var/www/html/creaprint

do you have any ide whats wrong with it?

thanks!

**Comment by Vaadasch** [comment]
Yes, a lot of things.

First of all, NEVER give public Internet-class IP addresses to your environment.
Use PRIVATE ip addresses : 
Class A: 10.0. 0.0 to 10.255. 255.255.
Class B: 172.16. 0.0 to 172.31. 255.255.
Class C: 192.168. 0.0 to 192.168. 255.255.
Thankfully, you didn't give your VM an IP address like 8.8.8.8 ...

If you tell nginx that location "domain.com/" must redirect to "server/creaprint/", why do you try to access to "domain.com/creaprint" ?

On your server 200.0.0.3, the folder /var/www/html/creaprint is served directly or is it /var/www/html that is served ? (by apache, probably)
If it is served directly, you don't need to specify your location, but if there is nothing else in html, you should put everything in it and serve it instead of serving creaprint/.
But without more description about your backend service, we can't provide more help. 
If your inside your own environment, try to set up your webserver without using Nginx Proxy Manager to first confirm that you control this part.
